sdcm/nemesis.py

```diff
@@ -1858,6 +1853,31 @@ def disrupt_corrupt_then_scrub(self):
         self.target_node.stop_scylla_server(verify_up=False, verify_down=True)
         self.target_node.start_scylla_server(verify_up=True, verify_down=False)
 
+    def disrupt_grow_shrink_cluster(self):
+        add_nodes_number = self.tester.params.get('add_node_cnt')
+
+        self._set_current_disruption("GrowCluster")
+        self.log.info("Start grow cluster on %s nodes", add_nodes_number)
+        for _ in range(add_nodes_number):
+            self.metrics_srv.event_start('add_node')
+            added_node = self._add_and_init_new_cluster_node()
+            added_node.running_nemesis = None
+            self.metrics_srv.event_stop('add_node')
+            self.log.info("New node added %s", added_node.name)
+        self.log.info("Finish cluster grow")
+
+        time.sleep(self.interval)
+
+        self._set_current_disruption("ShrinkCLuster")
+        self.log.info("Start shrink cluster on %s nodes", add_nodes_number)
+        for _ in range(add_nodes_number):
+            self.set_target_node()
+            self.log.info("Next node will be removed %s", self.target_node)
+            self.metrics_srv.event_start('del_node')
+            self.disrupt_nodetool_decommission(add_node=False)
```

Comment:
calling the disrupt function cause Decommission event to be raised, 
It's a bit confusing seeing all those different events for this one nemesis
there are 3 events being raised for this one nemesis